packages/style-loader/README.md

```diff
@@ -243,12 +213,13 @@ ready to make changes.
 
 ### Rules
 
-The following is a list of default rules and their identifiers which can be overridden:
+The following is a list of default rules/oneOfs and their identifiers which can be overridden:
```

Comment:
Could you update the rules section for the `@neutrinojs/web` readme with these changes too? :-)

Reply:
I updated it, but only referenced the top-level `style` rule and referred to `@neutrinojs/style-loader`. If you think I should duplicate all the oneOf details here, I can.